### PR TITLE
Update ExpressVPN 2FA status

### LINKED ISSUE #3521: ExpressVPN has two factor authentication now
Hello! I just created my Github account! I just wanna say now when I try to login, every time I get a prompt to input the code sent to my email. I weirdly haven't seen an official announcement, but they do now have two factor authentication, email based anyways!

### PR DESCRIPTION
**Pull request by Carlgo11**
Closes #3521

### CODE REVIEW

**Review by stephengroat — APPROVED** [review]

**Comment by Evernow** [comment]
No update? ExpressVPN still shows on the website that it doesn't support two factor authentication

**Comment by Carlgo11** [comment]
@evernow the PR isn’t merged yet and so it won’t show up on the site. We’re waiting for another approval before it can be merged.

**Review by brycx — APPROVED** [review]
Sorry for the delay @Evernow.

**Pull request merged**
